memory/src/main/scala/filodb.memory/format/vectors/HistogramVector.scala

```diff
@@ -379,7 +386,8 @@ class Appendable2DDeltaHistVector(factory: MemFactory,
  */
 class AppendableSectDeltaHistVector(factory: MemFactory,
                                     vectPtr: Ptr.U8,
-                                    maxBytes: Int) extends AppendableHistogramVector(factory, vectPtr, maxBytes) {
+                                    maxBytes: Int)
+extends AppendableHistogramVector(factory, vectPtr, maxBytes) with StrictLogging {
```

Comment:
Move logging to companion object  since we dont want as many logger objects as vectors.